File: sdks/cpp/tests/scaffold_test.cpp
#include <catch2/catch_test_macros.hpp>
#include <string>
#include "swissarmynoife/sdk_info.hpp"

TEST_CASE("sdk info name", "[scaffold]") {
  REQUIRE(std::string(swissarmynoife::SdkInfo::name) == "swissarmynoife-sdk-cpp");
}

File: sdks/php/tests/ScaffoldTest.php
<?php

declare(strict_types=1);

namespace SwissArmyNoife\Sdk\Tests;

use PHPUnit\Framework\TestCase;
use SwissArmyNoife\Sdk\SdkInfo;

final class ScaffoldTest extends TestCase
{
    public function testSdkInfoName(): void
    {
        $this->assertSame('swissarmynoife/sdk', SdkInfo::NAME);
    }
}
